projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -100,6 +100,7 @@ logging.level.io.swagger.models.parameters.AbstractSerializableParameter=ERROR
 datajobs.control.k8s.namespace=
 datajobs.control.k8s.kubeconfig=${HOME}/.kube/config
 datajobs.control.k8s.k8sSupportsV1CronJob=false
+datajobs.control.k8s.jobTTLAfterFinished=3600
```

Comment:
Sorry I forgot: please add suffix with the unit. jobTTLAfterFinishedSeconds
It's important to make clear what is the unit otherwise we do not know is it minutes, seconds, hours ...

Reply:
gotcha

Reply:
Thanks